shell/components/FixedBanner.vue

```diff
@@ -168,16 +168,20 @@ export default {
     >
       <!-- text as array to support line breaks programmatically rather than just exposing HTML -->
       <div v-if="isTextAnArray">
-        <p
+        <div
```

Comment:
The `<p/>` styles that were set above by the parent were getting overwritten by a global style https://github.com/rancher/dashboard/blob/master/shell/assets/styles/base/_typography.scss#L38-L42